json/unbound.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "Unbound",
+    "slug": "unbound",
+    "categories": [
+        13
+    ],
+    "date_created": "2024-11-27",
+    "type": "ct",
+    "updateable": true,
+    "privileged": false,
+    "interface_port": "5335",
+    "documentation": null,
+    "website": "https://www.nlnetlabs.nl/projects/unbound/about/",
+    "logo": "https://nlnetlabs.nl/static/logos/Unbound/Unbound_FC_Shaded_cropped.svg",
+    "description": "Unbound is a validating, recursive, caching DNS resolver. It is designed to be fast and lean and incorporates modern features based on open standards.",
+    "install_methods": [
+        {
+            "type": "default",
+            "script": "ct/unbound.sh",
+            "resources": {
+                "cpu": "1",
+                "ram": "512",
+                "hdd": "2",
+                "os": "debian",
+                "version": "12"
+            }
+        }
+    ],
+    "default_credentials": {
+        "username": null,
+        "password": null
+    },
+    "notes": [
+        {
+            "text": "Unbound listens to DNS requests at port 5335",
```

Comment:
We can remove this note. port is already shown through `interface_port`

Reply:
Fixed.